mobile/cmp/popover/Popover.js

```diff
@@ -0,0 +1,191 @@
+            popper = usePopper(impl.targetEl, impl.contentEl, {
+                placement: impl.menuPositionToPlacement(position),
+                strategy: 'fixed',
+                modifiers: [
+                    {name: 'preventOverflow', options: {padding: 10}}
+                ],
+                ...popperOptions
+            });
+
+        // Respect controlled mode prop
+        if (!isNil(isOpen)) {
+            impl.setControlledMode(isOpen, onInteraction);
+        }
+
+        return div({
+            className,
+            items: [
+                div({
+                    ref: impl.targetRef,
+                    className: 'xh-popover__target-wrapper',
+                    item: elementFromContent(target),
+                    onClick: () => impl.toggleOpen()
+                }),
+                div({
+                    ref: impl.contentRef,
+                    omit: !impl.isOpen,
+                    style: popper?.styles?.popper,
+                    className: 'xh-popover__content-wrapper',
+                    items: elementFromContent(content)
+                }),
+                div({
+                    omit: !impl.isOpen,
+                    className: classNames(
+                        'xh-popover__content-overlay',
+                        withMask ? 'xh-popover__content-overlay--mask' : null
+                    ),
+                    onClick: () => impl.setIsOpen(false)
+                })
+            ]
+        });
+    }
+});
+
+Popover.propTypes = {
+    /** Component to display inside the popover */
+    content: PT.oneOfType([PT.element, PT.object, PT.func]),
+
+    /** The target to which the popover content is attached */
+    target: PT.oneOfType([PT.element, PT.object, PT.func]),
+
+    /** Whether the popover is visible. Passing this prop puts the popover in controlled mode */
+    isOpen: PT.bool,
+
+    /**
+     * Callback invoked in controlled mode when the popover open state _would_ change due to user interaction.
+     * Receives (nextOpenState: boolean)
+     */
+    onInteraction: PT.func,
+
+    /** Whether to display a mask behind the popover */
+    withMask: PT.bool,
+
+    /** The position (relative to the target) at which the popover should appear. Default 'auto' */
+    position: PT.oneOf([
+        'top-left', 'top', 'top-right',
+        'right-top', 'right', 'right-bottom',
+        'bottom-right', 'bottom', 'bottom-left',
+        'left-bottom', 'left', 'left-top',
+        'auto'
+    ]),
+
+    /** Escape hatch to provide additional options to the PopperJS implementation */
+    popperOptions: PT.object
+};
+
+class LocalModel extends HoistModel {
+
+    targetRef = createObservableRef();
+    contentRef = createObservableRef();
+    @observable isOpen;
+
+    _onInteraction
+    _controlledMode = false;
+
+    get targetEl() {
+        return ReactDom.findDOMNode(this.targetRef.current);
+    }
+
+    get contentEl() {
+        return this.contentRef.current;
```

Comment:
These two `xxxEl` getters are different in their implementation - why is that?

Reply:
That was cruft from the previous implementation, where target was an onsen button and the ref would return the Component rather than the element. Now that targetEl is the wrapper div, its no longer needed - will clean up. Thanks for catching! 👍